functionary/ui/templates/core/package_detail.html

```diff
@@ -1,52 +1,40 @@
-{% extends "base_bulma.html" %}
+{% extends "base.html" %}
 {% block content %}
     {% with package.active_functions as functions %}
-        <div class="column is-full">
-            <nav class="breadcrumb has-arrow-separator" aria-label="breadcrumbs">
-                <ul>
-                    <li>
-                        <a href="{% url 'ui:package-list' %}">Package List</a>
-                    </li>
-                    <li class="is-active">
-                        <a href="#">{{ package.render_name }}</a>
-                    </li>
-                </ul>
-            </nav>
-        </div>
-        <div class="column">
-            <div class="block">
-                <h1 class="title is-1">
-                    <p>
-                        <i class="fa fa-cubes"></i>
-                        <span>{{ package.render_name }}</span>
-                    </p>
-                </h1>
-            </div>
-            <div class="column has-addons">
-                {% if package.description %}
-                    <div class="field">
-                        <label class="label" for="desc">Description:</label>
-                        <span id="desc" class="ml-4">{{ package.description }}</span>
-                    </div>
-                {% elif package.summary %}
-                    <div class="field">
-                        <label class="label" for="desc">Summary:</label>
-                        <span id="desc" class="ml-4">{{ package.summary }}</span>
-                    </div>
-                {% endif %}
-                <div class="field">
-                    <label class="label" for="funcs">
-                        <i class="fa fa-cube"></i>&nbsp;Functions:
-                    </label>
-                    <ul id="funcs">
+        <nav aria-label="breadcrumb">
+            <ol class="breadcrumb">
+                <li class="breadcrumb-item">
+                    <a class="text-decoration-none" href="{% url 'ui:package-list' %}">Package List</a>
+                </li>
+                <li class="breadcrumb-item active" aria-current="page">{{ package.render_name }}</li>
+            </ol>
+        </nav>
+        <h1 class="py-2">
+            <i class="fa fa-cubes fa-xl"></i>
+            {{ package.render_name }}
+        </h1>
+        <div class="vstack gap-4 py-2">
```

Comment:
Something about the spacing here feels off.  The Summary  or Description feels too close to the package name and then the space between Summary and Functions is a little too large.  Playing around with it, removing the `gap-4` here and changing the h1 to be `py-4` looks more evenly spaced, but play around with it.